__mocks__/react-native-ble-manager.ts

```
@@ -0,0 +1,123 @@
+// __mocks__/react-native-ble-manager.ts
+
+import { jest } from '@jest/globals'; // Use this import style with Jest >= 27
+
+// Mock the NativeEventEmitter
+const mockEmitter = new NativeEventEmitter();
+const emitEvent = (eventName: string, data: any) => {
```

ESLint

[error] 'emitEvent' is assigned a value but never used.

(@typescript-eslint/no-unused-vars)